Documentation/guides/messages/xa0122.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Xamarin.Android warning XA0122
+description: XA0122 warning code
+ms.date: 02/07/2020
+---
+# Xamarin.Android warning XA0122
+
```

Comment:
This isn't actionable if you're the library author.  *What should you do* if you're the library author to avoid this message?

The answer: use the `@(AndroidExternalLibrary)` Build action *instead of* `[DoNotPackageAttribute]`.